onnx/backend/test/cpp/driver_test.cc

```diff
@@ -0,0 +1,249 @@
+#include "gtest_utils.h"
+
+#include "onnx/checker.h"
+#include "onnx/onnxifi.h"
+#include "onnx/onnxifi_loader.h"
+#include "onnx/string_utils.h"
+
+#ifndef ONNXIFI_DUMMY_BACKEND
+#define ONNXIFI_DUMMY_BACKEND false
+#endif
+
+const float ONNXIFI_TESTDATA_EPS = 1e-5;
+
+template <typename T>
+class CompareOnnxifiData {
+ public:
+  bool IsEqual(void* P, void* Q) {
```

Comment:
Shall we add a parameter `eps`, and make ONNX)FI_TESTDATA_EPS as default value?

Also feel this is quite useful, probably lift it util?

Reply:
@houseroad For the eps I think you are right, I will do it. As for IsEqual, I think it is fine to leave it here now. If there is other scenario gonna use this in the future, we can do it then. 

In fact, I am thinking of building a whole operator system for onnxifi backend, since I dont think we have now...

Reply:
I am about to do it in the python world (seriously), are you interested in ?

Reply:
Sure, glad if I can help